platforms/ios/src/TangramMap/TGMapData.h

```diff
@@ -14,7 +14,9 @@
 /**
  Dictionary of feature properties keyed by their property name
  */
-typedef NSDictionary<NSString *, NSString *> FeatureProperties;
+typedef NSDictionary<NSString *, NSString *> TGFeatureProperties;
+
+@class TGMapViewController;
```

Comment:
Clang is smart enough to work around some of the forward declaration annoyances which can then result in things occasionally not working either. I think the rule is: If wherever `TGMapData` gets used for example also was importing `TGMapViewController.h` then clang will resolve the dependency that way.